Source/Core/DolphinQt/Main.cpp

```diff
@@ -101,27 +101,11 @@ int main(int argc, char* argv[])
   QApplication app(argc, argv);
 
 #ifdef _WIN32
-  // Get the default system font because Qt's way of obtaining it is outdated
-  NONCLIENTMETRICSW metrics = {};
-  LOGFONTW& logfont = metrics.lfMenuFont;
-  metrics.cbSize = sizeof(metrics);
-
-  if (SystemParametersInfoW(SPI_GETNONCLIENTMETRICS, sizeof(metrics), &metrics, 0))
-  {
-    // Sadly Qt 5 doesn't support turning a native font handle into a QFont so this is the next best
-    // thing
-    QFont font = QApplication::font();
-    font.setFamily(QString::fromStdWString(logfont.lfFaceName));
-
-    font.setItalic(logfont.lfItalic);
-    font.setStrikeOut(logfont.lfStrikeOut);
-    font.setUnderline(logfont.lfUnderline);
-
-    // The default font size is a bit too small
-    font.setPointSize(QFontInfo(font).pointSize() * 1.2);
-
-    QApplication::setFont(font);
-  }
+  // On Windows, Qt's default system font (MS Shell Dlg 2) is outdated.
+  // Interestingly, the QMenu font is correct and probably comes from lfMenuFont
+  // (Segoe UI on English computers).
+  // So use it for the entire application.
+  QApplication::setFont(QApplication::font("QMenu"));
```

Comment:
Please add a line about this being obsolete once we switch to [Qt 6 (which will release this year)](https://github.com/qt/qtbase/blob/35262678c31a4aea0df1516c565cf9a2e808b369/src/platformsupport/fontdatabases/windows/qwindowsfontdatabasebase.cpp#L760) (You don't need to add this link).

Reply:
New commit, or amend and force push?

Reply:
amend and force push please :)